ingest/exporter/ingestexportservice.py

```diff
@@ -23,8 +23,8 @@
 DEFAULT_DSS_URL = os.environ.get('DSS_API', 'http://dss.dev.data.humancellatlas.org')
 
 ERROR_TEMPLATE = {
```

Comment:
I see that there is a `util` package in `ingest/`. You could consider creating a new class there with two class variables: type and title. Then you could import the class both here and in importer.py and that way you ensure that the code won't ever be out of sync.